src/detections/rule/matchers.rs

```diff
@@ -2033,6 +2052,44 @@ mod tests {
         check_select(rule_str, record_json_str, false);
     }
 
+    #[test]
+    fn test_wildcard_converted_ends_with_shorter_val_notedetect() {
```

Comment:
Is this duplicated following test? 
https://github.com/Yamato-Security/hayabusa/pull/890/commits/67f322f9991e6b590e8e05ead249977569156fc3#diff-23d05016e0ab603d71cd4137b92fafbd2178d6cca1068e619342c10613cf68cbR1980 ?

Reply:
@hitenkoku 
Thank you so much for detail review !
It was a test case intended to cover the following two  different `false` conditions.

`fn test_wildcard_converted_starts_with_shorter_val_notdetect()`
https://github.com/Yamato-Security/hayabusa/blob/67f322f9991e6b590e8e05ead249977569156fc3/src/detections/rule/matchers.rs#L387-L391

`fn test_wildcard_converted_ends_with_shorter_val_notdetect()`
https://github.com/Yamato-Security/hayabusa/blob/67f322f9991e6b590e8e05ead249977569156fc3/src/detections/rule/matchers.rs#L398-L403

Reply:
I'm sorry, I understand!  The test below is a complete duplicate and will be removed!

https://github.com/Yamato-Security/hayabusa/blob/67f322f9991e6b590e8e05ead249977569156fc3/src/detections/rule/matchers.rs#L2037-L2053

Reply:
I removed duplicate test in below commit !
https://github.com/Yamato-Security/hayabusa/pull/890/commits/70a81a69c3f36e1ff1061d8e998d28bc9053198c